modules/tasks/tasks/src/main/resources/webapp/js/directives.js

```diff
@@ -428,12 +434,16 @@
                 });
 
                 element.bind('blur', function (event) {
+                    var content;
                     event.stopPropagation();
                     if(element[0] !== event.target){
                         return;
                     }
-                    ngModel.$setViewValue(readContent(element));
-                    scope.$apply();
+                    content = readContent(element);
+                    if (!(attrs.divPlaceholder && parent.msg(attrs.divPlaceholder).toLowerCase() === content.toLowerCase())) {
```

Comment:
why is the placeholder considered content? The proper fix would be to block this behaviour, the placeholder shouldn't be treated as input